consensus/polybft/fsm.go

```diff
@@ -442,10 +450,8 @@ func (f *fsm) VerifyStateTransactions(transactions []*types.Transaction) error {
 
 // Insert inserts the sealed proposal
 func (f *fsm) Insert(proposal []byte, committedSeals []*messages.CommittedSeal) (*types.Block, error) {
-	newBlock := &types.Block{}
-	if err := newBlock.UnmarshalRLP(proposal); err != nil {
-		return nil, fmt.Errorf("cannot unmarshal proposal: %w", err)
-	}
+	newBlock := f.target.Block
```

Comment:
Nitpick: can we create the FullBlock here and work with it forward?

Reply:
I do not think so because the full block was created already as part of Insert and Validate.